plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/FileManagementUtil.java

```diff
@@ -0,0 +1,399 @@
+            } else {
+                FileManagementUtil.copy(srcPath, dstPath);
+            }
+        }
+    }
+
+    public static List getAllFilesPresentInFolder(File srcPath) {
+        List fileList = new ArrayList();
+        if (srcPath.isDirectory()) {
+            String files[] = srcPath.list();
+            for (int i = 0; i < files.length; i++) {
+                fileList.addAll(getAllFilesPresentInFolder(new File(srcPath, files[i])));
+            }
+        } else {
+            fileList.add(srcPath.getAbsolutePath());
+        }
+        return fileList;
+    }
+
+    public static void removeEmptyDirectories(File srcPath) {
+        if (srcPath.isDirectory()) {
+            String files[] = srcPath.list();
+            for (int i = 0; i < files.length; i++) {
+                removeEmptyDirectories(new File(srcPath, files[i]));
+            }
+            if (srcPath.list().length == 0) {
+                srcPath.delete();
+            }
+        }
+    }
+
+    static public void zipFolder(String srcFolder, String destZipFile) throws Exception {
+        ZipOutputStream zip = null;
+        FileOutputStream fileWriter = null;
+        try {
+            fileWriter = new FileOutputStream(destZipFile);
+            zip = new ZipOutputStream(fileWriter);
+        } catch (FileNotFoundException e) {
+            logger.log(Level.FINE, HumantaskEditorConstants.ERROR_CREATING_CORRESPONDING_ZIP_FILE, e);
+        }
+        addFolderContentsToZip(srcFolder, zip);
+        try {
+            zip.flush();
+            zip.close();
+        } catch (IOException ex) {
+            logger.log(Level.FINE, HumantaskEditorConstants.ERROR_CREATING_CORRESPONDING_ZIP_FILE, ex);
+        }
+    }
+
+    static private void addToZip(String path, String srcFile, ZipOutputStream zip) {
+
+        File folder = new File(srcFile);
+
+        if (folder.isDirectory()) {
+            addFolderToZip(path, srcFile, zip);
+        } else {
+            // Transfer bytes from in to out
+            if (!srcFile.equals(".project")) {
+                byte[] buf = new byte[1024];
+                int len;
+                try {
+                    FileInputStream in = new FileInputStream(srcFile);
+                    String location = folder.getName();
+                    if (!path.equalsIgnoreCase("")) {
+                        location = path + File.separator + folder.getName();
+                    }
+                    zip.putNextEntry(new ZipEntry(location));
+                    while ((len = in.read(buf)) > 0) {
+                        zip.write(buf, 0, len);
+                    }
+                    in.close();
+                } catch (FileNotFoundException ex) {
+                    logger.log(Level.FINE, HumantaskEditorConstants.ERROR_CREATING_CORRESPONDING_ZIP_FILE, ex);
+                } catch (IOException e) {
+                    logger.log(Level.FINE, HumantaskEditorConstants.ERROR_CREATING_CORRESPONDING_ZIP_FILE, e);
```

Comment:
We are logging the exception and continuing forward. Why are we not throwing it?